### PR TITLE
[MI-2420] Add test cases for command.go

### PR DESCRIPTION
**Pull request by ayusht2810**

### CODE REVIEW

File: server/plugin/command_test.go [hunk 1]
```diff
@@ -415,6 +415,34 @@ func TestHandleSubscriptions(t *testing.T) {
...
+		UserId: testutils.GetID(),
+	}
+	for _, testCase := range []struct {
+		description      string
+		params           []string
+		expectedResponse string
+	}{
+		{
+			description:      "HandleCreate: Invalid number of params",
+			expectedResponse: "Invalid create command. Available commands are 'incident' and 'request'.",
```
Comment (Kshitij-Katiyar) [inline server/plugin/command_test.go:430]: I think there will be a constant for this and if present use that

File: server/plugin/command_test.go [hunk 2]
```diff
@@ -415,6 +415,34 @@ func TestHandleSubscriptions(t *testing.T) {
...
+		},
+		{
+			description:      "HandleCreate: Unknown command",
+			params:           []string{"invalidCommand"},
+			expectedResponse: "Unknown subcommand invalidCommand",
```
Comment (Kshitij-Katiyar) [inline server/plugin/command_test.go:435]: same

File: server/plugin/command_test.go [hunk 3]
```diff
@@ -477,6 +505,68 @@ func TestHandleSearchAndShare(t *testing.T) {
 	}
 }
 
+func TestHandleCreateIncident(t *testing.T) {
+	p := Plugin{}
+	mockAPI := &plugintest.API{}
+	args := &model.CommandArgs{
+		UserId: testutils.GetID(),
+	}
+	for _, testCase := range []struct {
```
Comment (Kshitij-Katiyar) [inline server/plugin/command_test.go:514]: dont use loop if there is only 1 subtest

File: server/plugin/command_test.go [hunk 4]
```diff
@@ -477,6 +505,68 @@ func TestHandleSearchAndShare(t *testing.T) {
...
+		description   string
+		setupAPI      func(*plugintest.API)
+		expectedError string
+	}{
+		{
+			description: "HandleCreateRequest: Success",
```
Comment (Kshitij-Katiyar) [inline server/plugin/command_test.go:551]: same

File: server/plugin/command_test.go (hunk 1 above)
Comment (ayusht2810) [reply server/plugin/command_test.go:431]: no there is no constant for this

File: server/plugin/command_test.go (hunk 2 above)
Comment (ayusht2810) [reply server/plugin/command_test.go:436]: same

File: server/plugin/command_test.go (hunk 3 above)
Comment (ayusht2810) [reply server/plugin/command_test.go]: this test case is removed now

File: server/plugin/command_test.go (hunk 4 above)
Comment (ayusht2810) [reply server/plugin/command_test.go]: same

**Review by Kshitij-Katiyar — APPROVED** [review]

**Pull request merged**